pynitrokey/fido2/client.py

```diff
@@ -347,6 +347,10 @@ def simple_secret(
 
         return output
 
+    def has_pin_set(self) -> bool:
```

Comment:
nit: I think this should either be `is_pin_set` or `has_pin`.

Reply:
good one, was thinking about this far too long :D ... took `has_pin`